accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -0,0 +1,627 @@
+select the button Add debit note and select the first option in the wizard: *1: Anula 
+Documentos de referencia.*
+
+.. image:: media/Chile33.png
+   :align: center
+
+
+Vendor Bills
+------------
+As part of the Chilean localization, you can configure your Incoming email server as the same you
+have register in the SII in order to:
+
+- Automatically receive the vendor bills DTE and create the vendor bill based on this information. 
+- Automatically Send the reception acknowledgement to your vendor. 
+- Accept or Claim the document and send this status to your vendor. 
+
+
+Reception
+~~~~~~~~~
+As soon as the vendor email with the attached DTE is received:
+1. The vendor Bill mapping all the information included in the xml.
+2. An email is sent to the vendor with the Reception acknowledgement. 
+3. The DTE status is set as:  Acuse de Recibido Enviado
+
+.. image:: media/Chile34.png
+   :align: center
+
+
+Acceptation
+~~~~~~~~~~~
+If all the commercial information is correct on your vendor bill, you can accept the document using
+the button: Aceptar Documento, once this is done the DTE Acceptation Status  changes to: Accepted and
+an email of acceptance is sent to the vendor. 
+
+.. image:: media/Chile35.png
+   :align: center
+
+Claim
+~~~~~
+In case there is a commercial issue or the information is not correct on your vendor bill, you can
+Claim the document before validating it,  using the button: Claim, once this is done the DTE
+Acceptation Status  change to: Claim  and an email of acceptance is sent to the vendor. 
+
+.. image:: media/Chile36.png
+   :align: center
+
+If you claim a vendor bill the status changes from draft to cancel automatically, considering this
+as best practice, all the Claim documents should be cancelled as they won’t be valid for your
+accounting records. 
+
+Financial Reports
+=================
+
+Balance Tributario de 8 Columnas
+--------------------------------
+This report presents the accounts in detail (with their respective balances), classifying them
+according to their origin and determining the level of profit or loss that the business had within
+the evaluated period of time, so that a real and complete knowledge of the status of a company. 
+
+You can find this report in :menuselection:`Accounting --> Accounting --> Reports`
+
+.. image:: media/Chile37.png
+   :align: center
+
+Propuesta F29
+-------------
+The form F29 is a new system that the SII enabled to taxpayers, and that replaces the Purchase and
+Sales Books. This report is integrated by Purchase Register (CR) and the Sales Register (RV)
+its purpose is to support the transactions realted to VAT improving its control and declaration. 
+
+This record is supplied by the electronic tax documents (DTE’s) that have been received by the
+SII.
+
+You can find this report in :menuselection:`Accounting --> Accounting --> Reports`
+
+.. image:: media/Chile38.png
+   :align: center
+
+
+
```

Comment:
[Seealso section?](https://www.odoo.com/documentation/user/13.0/contributing/documentation/rst_cheat_sheet.html#seealso)

Reply:
> Hello!
> Good job with the rst 👍
> 
> * Ideally, please add _alt_ tags to your images as explained [here](https://www.odoo.com/documentation/user/13.0/contributing/documentation/content_guidelines.html#alt-tags).
> * Ideally, leave a blank line after all titles. For example:
> 
> ```
> Title 1
> =======
> 
> Paragraph here.
> ```
> 
> * Also, could you check the end of some sentences? It seems that there are a lot of sentences being separated by a comma, rather than ending them with a point (e.g. "Sentence 1, sentence 2." instead of "Sentence 1. Sentence 2.")
> * Other comments are directly put on the code.
> * Abreviations such as _it's_ are rather for spoken language. Ideally, write them _it is_ instead.
> 
> Have a good day
> 
> Best regards,
> Jonathan

@jcs-odoo Thank you for the Feedback and the corrections,  I committed you suggestions (sorry  I was doing it individually at the beginning).  In the last commit with I added the Alt information on the images (except from img 16 to 20, as I was having issues with the alt on the nested list, it wasn't working), I also added the blank line after the subtitles. 

Regards.

Reply:
@jcs-odoo sorry but I don't get in this section it might add the Seealso section.